src/Writer/ExtensionPluginManager.php

```diff
@@ -14,12 +16,15 @@
 use function sprintf;
 use function substr;
 
+// phpcs:disable Generic.Files.LineLength.TooLong
```

Comment:
Disabled for the entire class?

Reply:
Yeh, waste of time to hunt these down in a file like this 😁